app/core/migrations/0004_auto_20210413_2228.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.1.15 on 2021-04-13 22:28
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
No problem since this is an exercise, but normally we squash migrations into just one when developing a new feature to keep the number of migrations low. You can do that by deleting all migrations you created here and running `makemigrations` again.

Reply:
Good to know. Thank you!